src/test/java/hudson/plugins/active_directory/ActiveDirectoryDomainTest.java

```diff
@@ -0,0 +1,66 @@
+package hudson.plugins.active_directory;
+
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.jvnet.hudson.test.FlagRule;
+
+import hudson.util.FormValidation;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+
+public class ActiveDirectoryDomainTest {
```

Comment:
These tests are explicitly for FiPS so it should be noted in the class name for easier future investigations.